This patch adds a pre-generated configs/kernel-driver.yaml. Normally,
it's generated as defined by configs/Makefile.am. The generation
depends on running modinfo(1) which seemingly does not work in the
pbuilder environment.

To re-generate the file:

  cd configs; ../tools/lirc-kerneldriver ./kernel-driver.dist \
      > kernel-driver.yaml

The generated file reflects the current kernel and the modules therein.

--- /dev/null
+++ b/configs/kernel-drivers.yaml
@@ -0,0 +1,36 @@
+#
+# Created by lirc-kerneldrivers at Tue Oct 13 14:58:04 2015
+#
+drivers:
+
+    lirc_sasem:
+        can_send:           True
+        device_hint:        /dev/lirc*
+        type:               mode2
+        id:                 sasem
+        modinit:            modprobe lirc_sasem
+
+    ati_remote:
+        can_send:           False
+        device_hint:        /dev/lirc*
+        type:               code
+        modinit:            modprobe ati_remote
+
+    lirc_sir:
+        can_send:           True
+        device_hint:        /dev/lirc*
+        type:               mode2
+        id:                 sir
+        modinit:            modprobe lirc_sir
+
+    lirc_bt829:
+        can_send:           False
+        device_hint:        /dev/lirc*
+        type:               code
+        modinit:            modprobe lirc_bt829
+
+    lirc_serial:
+        can_send:           True
+        device_hint:        /dev/lirc*
+        type:               mode2
+        modinit:            modprobe lirc_serial
